pkg/kubectl/cmd/apply/apply_test.go

```diff
@@ -68,6 +69,12 @@ var (
 	codec = scheme.Codecs.LegacyCodec(scheme.Scheme.PrioritizedVersionsAllGroups()...)
 )
 
+func init() {
+	klog.InitFlags(nil)
+
+	cmdtesting.InitTestErrorHandler()
```

Comment:
Does it have to be initialized in `init` rather than each method?

Reply:
if you initialize it in each method, given it's setting the global variable, seems like a race to me